tasks/setup_requirements.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: Install libselinux-python package for 'copy' task
+  yum:
+    name: libselinux-python
+    state: present
+  when: ansible_os_family == "RedHat"
```

Comment:
Since this is a role, we have no control over `gather_facts` which is specified at the play level. However, `gather_facts` is simply a convenience for the `setup` module. Try explicitly invoking the `setup` module before you pivot on the `ansible_os_family` variable.